akka-projection-slick/src/test/scala/akka/projection/slick/SlickProjectionSpec.scala

```diff
@@ -170,13 +173,25 @@ class SlickProjectionSpec extends SlickSpec(SlickProjectionSpec.config) with Any
 
   val logger: Logger = LoggerFactory.getLogger(this.getClass)
 
+  val dbConfig: DatabaseConfig[H2Profile] = DatabaseConfig.forConfig(SlickSettings.configPath, config)
+
+  val offsetStore = new SlickOffsetStore(dbConfig.db, dbConfig.profile, SlickSettings(system))
+
+  val projectionTestKit = new ProjectionTestKit(testKit)
+
+  override protected def afterAll(): Unit = {
+    super.afterAll()
+    dbConfig.db.close()
+  }
   val repository = new TestRepository(dbConfig)
 
   implicit val actorSystem: ActorSystem[Nothing] = testKit.system
   implicit val executionContext: ExecutionContext = testKit.system.executionContext
 
   override protected def beforeAll(): Unit = {
     super.beforeAll()
+    // create offset table
+    Await.ready(offsetStore.createIfNotExists, 3.seconds)
```

Comment:
it's almost always better to use Await.result, because `ready` just returns a Boolean and hides any failures

Reply:
we have been using ready. Will change.